### PR TITLE
add playback related changes to the release guide

### PR DESCRIPTION
**Pull request by mat-hek**

### CODE REVIEW

File: guides/upgrading/v0.11.md [hunk 1]
```diff
@@ -14,11 +14,125 @@ defp deps do
...
+```
+
+Rename `handle_prepared_to_playing/2` to `handle_playing/2`
+
+```diff
+- def handle_prepared_to_playing(ctx, state) do
++ def handle_playing(ctx, state) do
+```
+
+Remove `handle_playing_to_prepared/2`, `handle_prepared_to_stopped/2` and `handle_shutdown/2`. Instead, use `Membrane.ResourceGuard`, `Membrane.UtilitySupervisor` or `handle_terminate_request/2`ą
```
Comment (varsill) [inline guides/upgrading/v0.11.md:67]:
```suggestion
Remove `handle_playing_to_prepared/2`, `handle_prepared_to_stopped/2` and `handle_shutdown/2`. Instead, use `Membrane.ResourceGuard`, `Membrane.UtilitySupervisor` or `handle_terminate_request/2`:
```

File: guides/upgrading/v0.11.md [hunk 2]
```diff
@@ -14,11 +14,125 @@ defp deps do
...
+```diff
+- def handle_stopped_to_prepared(_ctx, state) do
+-  resource = create_resource()
+-  {:ok, %{state | resource: resource}}
+- end
+- ...
+- def handle_prepared_to_stopped(_ctx, state) do
+-  cleanup_resource(state.resource)
+- end
++ def handle_setup(ctx, state) do
```
Comment (varsill) [inline guides/upgrading/v0.11.md:78]: How about a brief description of when to use `handle_init/2` and when to use `handle_setup/2`?

File: guides/upgrading/v0.11.md [hunk 3]
```diff
@@ -14,11 +14,125 @@ defp deps do
...
++  resource = create_resource()
++  # The resource will be automatically cleaned up when the component terminates
++  # or you can do that manually with `Membrane.Resource.cleanup_resource/2`
```
Comment (varsill) [inline guides/upgrading/v0.11.md:81]: Are you sure that this sentence is saying what it is meant to say? :D 
I am not sure if that `or` statement fits in the place.

File: guides/upgrading/v0.11.md [hunk 4]
```diff
@@ -74,7 +186,7 @@ Option `:accepted_format` is able to receive:
 
 ## Update options definitions
 
-Remove `:type` key and related value from defined options. Add `:spec` instead, if it haven't been added before, and proper `:inspector`, if option has default value, that shouldn't be inspected by `inspect/1` during generating docs.
+Remove `:type` key and related value from defined options. Add `:spec` instead, if it hasn't been added before, and proper `:inspector`, if the option has a default value, that shouldn't be inspected by `inspect/1` during generating docs.
```
Comment (varsill) [inline guides/upgrading/v0.11.md:189]:
```suggestion
Remove `:type` key and related value from the options definitions. Add `:spec` instead, if it hasn't been added before, and proper `:inspector`, if the option has a default value, that shouldn't be inspected by `inspect/1` during generating docs.
```

**Review by varsill — CHANGES_REQUESTED** [review]

**Review by FelonEkonom — APPROVED** [review]

File: guides/upgrading/v0.11.md (hunk 2 above)
Comment (mat-hek) [reply guides/upgrading/v0.11.md:82]: done

**Review by varsill — APPROVED** [review]

**Pull request merged**